Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml

```
@@ -321,6 +337,10 @@
           Quantity: 5
         - ReagentId: DoctorsDelight
           Quantity: 6
+  - type: Tag
+    tags:
+    - Pizza
```

yamllint

[error] wrong indentation: expected 6 but found 4

(indentation)